.changeset/spotty-panthers-accept.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-hive/cli': minor
+---
+
+Adds introspect command
```

Comment:
This could include an example usage 😇

Reply:
it's in readme!

Reply:
Then link to the readme from here 😇
My point is that the changelog is a great place to give an quick example, without people having to dig docs. As I said it is optional.